### DISCUSSION TITLE
Non-Expo Project

### DISCUSSION THREAD

**Discussion by kevinchristianto** [tamagui/tamagui#518] [Q&A] [answered]
Is is possible to use tamagui outside of expo project?
I'm installing tamagui in my bare react native project, and I got an error regarding expo-linear-gradient couldn't be found.

Should I install expo-linear-project and its dependencies (expo)?
Any answers would be appreciated! 🙌

<img width="509" alt="image" src="https://user-images.githubusercontent.com/19814559/211510953-8f5b684c-3878-44c8-a7c2-4a7dc4837d07.png">

**Comment by natew** [comment, marked as answer]
Yes it works in bare React Native, v1.0.5 no longer requires installing expo-linear-gradient but you can also do that in bare RN, docs are updated here too.

**Reply by kevinchristianto** [reply]
Great! Will update my tamagui to 1.0.5 now, thanks🙌

**Reply by natew** [reply]
Actually it does require install for Tamagui proper still, I guess Metro doesn’t support any form of lazy imports as of yet.

**Reply by kevinchristianto** [reply]
I see, is there any chance of migrating expo-linear-gradient to react-native-linear-gradient?
The cons of expo-linear-gradient is that it requires expo and expo modules, while react-native-linear-gradient just works out of the box.

**Reply by natew** [reply]
Didn’t know about it I’ll check it out. I’m also considering moving the package to be only importable by tamagui/linear-gradient since it’s the only package that brings along native deps.

**Reply by kevinchristianto** [reply]
Great, thanks for the support!
